inputfiles/overridingTypes.json

```diff
@@ -1822,5 +1822,13 @@
         "signatures": [
             "getAttributeNode(name: string): Attr | null"
         ]
+    },
+    {
+        "kind": "method",
+        "interface": "WebSocket",
+        "name": "send",
+        "signatures": [
+            "send(data: USVString | ArrayBuffer | Blob | ArrayBufferView): void;"
```

Comment:
signatures in `overridingTypes.json` doesn't need semicolons as they are automatically added.